openff/toolkit/topology/molecule.py

```diff
@@ -5329,6 +5363,44 @@ def from_mapped_smiles(
 
         return offmol.remap(adjusted_mapping, current_to_new=True)
 
+    @classmethod
+    @requires_package("mmic_openff")
+    def from_mmschema(cls, schema_object, version=None, **kwargs):
+        """
+        Returns a :class:`Molecule` from an MMElemental Molecule object.
+
+        Parameters
+        ----------
+        schema_object : mmelemental.models.Molecule
+            MMElemental molecule object that adheres to the MMSchema spec.
+
+        version : int, default=None
+            MMSchema version. Overrides schema_object.schema_version.
+
+        kwargs : dict
+            Additional keyword arguments to pass to the constructor.
+
+        Returns
+        ---------
+        openff.toolkit.topology.molecule.Molecule
+            An OpenFF molecule object.
+
+        Examples
+        --------
+        Create Molecule from MMElemental Molecule:
+
+        >>> import mmelemental
+        >>> mm_mol = mmelemental.models.Molecule(symbols=["C", "C"])
```

Comment:
(blocking) This is an invalid OpenFF molecule because it's not a valid kekule structure (doesn't have any bonds). It may not crash now, but it will eventually, so it may be best to do a molecule like water where we can easily fill in bonds.

Reply:
Docstrings updated [here](https://github.com/andrew-abimansour/openff-toolkit/commit/c76f1cddee4cb5f3c4c8b919d11b7b036c8ef35f#diff-68cb3f9873e00994ccf957cc089b18cbaf34302d5eeb90c34ca289803e10a69eR5393).